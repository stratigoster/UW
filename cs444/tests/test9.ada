PACKAGE foo IS
  b : BOOLEAN := true AND THEN NOT false;
  i : INTEGER := 3+4*5**(2**2);
  f : FLOAT := 2.0e-9;
  g : FLOAT := 10e-10000000000000000000000000000000000000000000000000;
  h : FLOAT := f * g - f + 10.0e2 & "FLOAT";
--                                ^^^^^^^^^^
-- type mismatch: cannot concatenate floats and strings
  s : STRING := "Bye World""" & "Hi Heaven"; 
END foo;
