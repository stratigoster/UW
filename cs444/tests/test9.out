***ERROR***:6:35:	Can only concatenate strings

------
Compilation error
------
